src/App.tsx

```diff
@@ -63,11 +39,12 @@ export const App: React.FC = () => {
                   </p>
                 )}
 
-                {author && !loaded && (
+                {statusOfUsersLoading === 'loading'
```

Comment:
beter to make enum for loading statuses

Reply:
still not fixed
<img width="725" alt="image" src="https://github.com/mate-academy/react_redux-list-of-posts/assets/70220667/b4270b40-0e79-476a-8ef5-4021389b240f">